src/recorder/session-recording.js

```diff
@@ -349,8 +349,9 @@ SessionRecording.prototype._sendRequest = function(currentReplayId, reqParams, r
             retryAfter: response.headers.get('Retry-After')
         });
     }.bind(this);
+    const apiHost = this._mixpanel.get_config('api_hosts')['record'] || 'https://api.mixpanel.com';
```

Comment:
Sorry for the delay getting back to this; you'd have to default to `getConfig('api_host')` otherwise you'll break non-US data residency. But why not be consistent with your other changes?
```suggestion
    const apiHost = this._mixpanel._getApiHost('record');
```

Reply:
No, it's better not to include dist files in a PR (it's all going to get rebuilt on a release candidate branch anyway)